enhancements/ingress/set-delete-http-headers.md

```diff
@@ -782,6 +765,26 @@ For this reason, the `host` header is in the deny list of headers that are prohi
 
 Please note that a custom `host` header may be specified via RouteSpec. Please refer to `Open Questions` for more details.
 
+### Cookie and Set-Cookie
+There are three reasons we blocked `Cookie` and `Set-Cookie`:
+
+1. There were no use cases mentioned for these headers in [RFE](https://issues.redhat.com/browse/RFE-464).
+
+2. The cookies that HAProxy sets are used for session tracking, to map a client connection to a particular 
```

Comment:
# 2 adds more meaning to the decision than # 1. I would swap the ordering.

Reply:
done